modules/webxr/webxr_interface_js.cpp

```diff
@@ -484,7 +473,17 @@ bool WebXRInterfaceJS::pre_draw_viewport(RID p_render_target) {
 Vector<BlitToScreen> WebXRInterfaceJS::post_draw_viewport(RID p_render_target, const Rect2 &p_screen_rect) {
 	Vector<BlitToScreen> blit_to_screen;
 
-	// We don't need to do anything here.
+	GLES3::TextureStorage *texture_storage = dynamic_cast<GLES3::TextureStorage *>(RSG::texture_storage);
```

Comment:
Don't we have a `GLES3::TextureStorage::get_singleton()` ?

Reply:
Whoa, yeah, for some reason I'm doing that cast through out this file. I'll update that to `GLES3::TextureStorage::get_singleton()` everywhere.